src/site/styles/variables.less

```diff
@@ -7,10 +7,11 @@
 @color-shadow-medium: #b0c0c0;
 @color-shadow-heavy: #aabbbb;
 @color-shadow-dark: #212835;
+@color-accent-color: #329A97;
 
-@font-body-family: "Segoe UI Semilight", "Segoe UI", "sans-serif";
+@font-body-family: "Segoe UI Semilight", "Segoe UI", "Lucida Grande", "sans-serif";
 @font-body-size: 1.17em;
-@font-fancy-family: "Segoe UI Light";
+@font-fancy-family: "Segoe UI Light", "Lucida Grande Narrow Light", "sans-serif";
```

Comment:
If it's for the font fallback issues,
1. That fix should be in its own PR
2. We want to have an actual font fallback for consistent fonts across browsers